Automattic/wp-calypso
# A4A: Implement the FAQs section on the new Migrations page.

This PR adds the FAQs section to the new Migrations page. **Please note that not all FAQs are included. I will update this later when we have the complete copies.**

<img width="1400" alt="Screenshot 2024-11-04 at 6 19 38 PM" src="https://github.com/user-attachments/assets/c16004fc-6f41-46a5-a628-01197cad82f7">



Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1380

## Proposed Changes

* Implement the FAQs section using the `FoldableFAQ` component.

## Why are these changes being made?

* This is part of the new Migrations page project.

## Testing Instructions

* Use the A4A live link and go to the `/migrations` page.
* Confirm you see the FAQs section with a set of questions.
* Confirm that expanding the question will add a hash fragment to the URL.
* Confirm that the hash fragment works correctly when refreshing the page.

## Pre-merge Checklist
